### PR TITLE
Added capability to specify scenarios in docker compose command.

### LINKED ISSUE #111: User should be able to specify specific scenarios to run in the reporting pipeline
Right now all scenarios will be run every time the reporting pipeline is invoked. Some configuration mechanism should be implemented to select the scenarios to be run.

### PR DESCRIPTION
**Pull request by Sorrento110**
# Description

* You can pass in scenarios by name as strings into the docker compose command to run only specific ones.
* Updated README.

Resolves #111 

To test:
- Run the regular command and check the post-seeding logs, there should be an info line that says : `"Running pipeline on all scenarios"`. Feel free to kill that process after this.
- Run this command: `docker compose run tests bash tests/run_tests.sh "SIDARTHE Equations" "SIR" ` and check the post-seeding logs. There should be an info line that says: `"Running pipeline on scenarios: ['SIDARTHE Equations', 'SIR']"`
- Ensure that the specified scenarios run and no others (SIDARTHE Equations and SIR scenarios.)

### CODE REVIEW

File: reporting/tests/report.py [hunk 1]
```diff
@@ -220,25 +221,28 @@ def do_task(url, task, kwargs={}):
...
-            generated_card = json.dumps(result["result"]["job_result"]["card"])
-            with open(ground_truth_path) as file:
-                truth = file.read()
-                files = {
-                    "test_json_file": generated_card,
-                    "ground_truth_file": truth,
-                }
-                eval = requests.post(
-                    f"{MIT_TR_URL}/evaluation/eval_model_card",
-                    params={"gpt_key": OPENAI_API_KEY},
```
Comment (brandomr) [inline reporting/tests/report.py:235]: what all changed here?

Comment (Sorrento110) [reply reporting/tests/report.py:235]: Additional error handling for an error I encountered.

File: reporting/tests/Dockerfile [hunk 2]
```diff
@@ -7,5 +7,4 @@ RUN pip install -r requirements.txt
 COPY scenarios scenarios/
 COPY tests tests/
 
-# CMD ["bash", "tests/run_tests.sh" , "&&", "while true; do sleep 1; done"]
 CMD [ "bash", "tests/run_tests.sh" ]
```
Comment (brandomr) [inline reporting/tests/Dockerfile:10]: If you change this to `ENTRYPOINT` then from the CLI the syntax can be the same:

```
docker compose run --build tests "SIDARTHE Equations" "SIR"
```

etc

**Review by brandomr — CHANGES_REQUESTED** [review]

File: reporting/tests/Dockerfile (hunk 2 above)
Comment (Sorrento110) [reply reporting/tests/Dockerfile:10]: Ah nice, I will update that since it's cleaner.

**Review by brandomr — APPROVED** [review]

**Pull request merged**